packages/support/src/Concerns/HasCellState.php

```diff
@@ -76,7 +76,7 @@ public function getState(): mixed
         }
 
         $state = ($this->getStateUsing !== null) ?
-            $this->evaluate($this->getStateUsing) :
+            $this->getStateUsing :
             $this->getStateFromRecord();
 
         if (is_string($state) && ($separator = $this->getSeparator())) {
```

Comment:
The breaking comment is correct, if the function returns a string that needs separating here or a blank value it would break